saleor/graphql/schema.graphql

```diff
@@ -5802,10 +6001,14 @@ type Weight {
 scalar WeightScalar
 
 enum WeightUnitsEnum {
-  KG
-  LB
-  OZ
   G
+  TONNE
+  OZ
+  LB
+  STONE
+  SHORT_TON
+  LONG_TON
```

Comment:
I don't think these are very useful. The long ton is obsolete, the short ton is hardly ever used.